security/rbac/scripts/create_login_properties.py

```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
```

Comment:
I'm not certain of the risks, but do you think it's worth being explict on the python version here?  Might prevent some oddities for users?
```suggestion
#!/usr/bin/env python3
```

Reply:
@rspurgeon concerns about tying it to python3:

1. What if user has python2 but not python3?
2. The script doesn't do anything complicated and I tested both python2 and python3.

So leaving as is.